app/javascripts/graphiql.jsx

```diff
@@ -6,6 +6,10 @@ import {AppConfig} from './config';
 import './css/graphiql.css'
 import 'graphiql/graphiql.css'
 
-const config = new AppConfig("graphiql")
-
-ReactDOM.render(<GraphiQLTool config={config} />, document.getElementById('graphiql-tool'));
\ No newline at end of file
+module.exports = {
+  bootstrap: function (bootstrapOptions = {}) {
+    this.bootstrapOptions = bootstrapOptions;
+    const config = new AppConfig("graphiql", bootstrapOptions);
+    ReactDOM.render(<GraphiQLTool config={config} />, document.getElementById('graphiql-tool'));
```

Comment:
Given that I was a bit optimistic and wanted to integrate this into Absinthe asap, I already published the 1.0.1 to jsdelivr and it's a pain to manually publish updates.

We can probably wait until the tool is properly versioned with a npm package to provide this.